sympy/concrete/summations.py

```diff
@@ -670,6 +673,95 @@ def is_absolutely_convergent(self):
         Sum.is_convergent()
         """
         return Sum(abs(self.function), self.limits).is_convergent()
+    @staticmethod
+    def is_expr_convergent(expr):
+        """
+        Checks for convergence of an expression.
+        Parameters
+        ----------
+        expr : TYPE
+            DESCRIPTION.
+
+        Raises
+        ------
+        TypeError
+            DESCRIPTION.
+
+        Returns
+        -------
+        bool
+            DESCRIPTION.
+
+        """
+        if isinstance(expr, (Mul,Add,Pow,Zero,Infinity)):
+            args = expr.args
+            a,b,c=0,0,0
+            if isinstance(expr ,Add):
+                for i in range(len(args)):
+                    if(isinstance(args[i],Sum)):
+                        if(args[i].is_convergent() == False):
+                            a+=1
```

Comment:
A space should be given on both sides of operators, and please give more descriptive names to variables

Reply:
Yeah I had given a thought about this while writing code , but as I couldn't find any suitable short variables names for these I named them a,b,c . Because something like 'a' stands for arguments which are not absolutely convergent but are conditionally convergent . Would adding comments describing a,b,c work ??